src/bit_set.rs

```diff
@@ -15,26 +15,21 @@ use std::slice;
 #[derive(Default,Debug)]
 pub struct Bitset {
     head: usize,
-    tail: Option<Box<Vec<usize>>>,
+    tail: Option<Vec<usize>>,
```

Comment:
Without the Box, Bitset takes up 4 words.  With it, Bitset takes up 2 words.  This is a _huge_ performance win, was something like 10%